go.mod

```diff
@@ -73,7 +73,7 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/vishvananda/netlink v1.1.1-0.20200603190939-5a869a71f0cb
 	github.com/vishvananda/netns v0.0.0-20191106174202-0a2b9b5464df
-	go.etcd.io/etcd v0.5.0-alpha.5.0.20200824191128-ae9734ed278b
+	go.etcd.io/etcd v0.5.0-alpha.5.0.20201125193152-8a03d2e9614b
```

Comment:
Isn't it possible to `go get` the version rather than the git sha so we'd have `go.etcd.io/etcd v3.4.14`? Looking at this line, it seems we're importing a very alpha version although we're not and we can't tell which version it is.

Reply:
@Rolinh no, `go get` will complain, I usually copy-paste the commit sha into the repository.